website/src/components/SearchPage/DownloadDialog/FieldSelector/FieldSelectorModal.tsx

```diff
@@ -0,0 +1,171 @@
+import { useState, type FC } from 'react';
```

Comment:
When selecting all fields, the option to copy the download URL is gone, because the URL probably got too long.

<img width="470" alt="image" src="https://github.com/user-attachments/assets/fa98faed-3528-4974-bacf-fff03b1c8e09" />

This is a regression, because it used to be possible.

It could still be implemented, by omitting the `fields` from the URL if _all_ fields are selected.

I can't say whether this is something to block on or not. If it isn't considered a blocker, could you kindly create a ticket for it and link it here?

Reply:
Thanks for flagging this - which I hadn't considered. IMO it's not something to block on because downloading all fields will be rare, and because generally this behaviour is better than the current behaviour which is super overwhelming. But it's a good suggestion https://github.com/loculus-project/loculus/issues/3870